InstallSpinnaker.sh

```diff
@@ -612,6 +623,10 @@ if [ "x$GOOGLE_PROJECT_ID" != "x" ]; then
   write_default_value "SPINNAKER_GOOGLE_PROJECT_ID" $GOOGLE_PROJECT_ID
 fi
 
+if [ "x$GOOGLE_CREDS_PATH" != "x" ]; then
```

Comment:
I'm not yet comfortable with this being here.

Also this is a little different than what we discussed.
deps_only is saying only to install the dependencies for spinnaker, but not do other stuff. I think that is legit. For example, developer builds execute that to prepare the host machine to run spinnaker, but spinnaker itself will never get installed because you are running it from source. There's no reason to add the other data installations (etc/default/spinnaker) either.

In your use case, you want to prepare both the host and the runtime data, but will be installing spinnaker separately. So I'd guard that (the data setup) with a separate flag. When you just say deps-only it should be off. Otherwise it should be on by default. But you could explicitly turn it on, so you can say deps-only +flag to meet your use case.